src/Controller/LoginController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\LoginManager;
+
+class LoginController extends AbstractController
+{
+    public function login()
+    {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $infos = array_map('trim', $_POST);
+            $infos = array_map('htmlentities', $infos);
+            $infos = array_map('stripslashes', $infos);
+            $login = new LoginManager();
+            if ((($login->checkUser($infos)) == true) && ($login->checkPass($infos)) == true) {
+                echo "okay" ;
+            } else {
+                echo "not ok";
+            }
+                var_dump($_POST);
+                var_dump($infos);
+                //header('location:/welcome');
```

Comment:
No ! Leaving a var_dump or dead code (commented code) in a PR is forbidden

Reply:
That's done with some other conflicts, no more dead code in LoginController. I paused the dev of the Login page. The var_dump() have been delete from the page.